src/NHibernate/NHibernate.csproj

```diff
@@ -41,12 +45,26 @@
     <PackageReference Include="Remotion.Linq.EagerFetching" Version="[2.1.0, 3.0)" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <Reference Include="System.ServiceModel" />
     <Reference Include="System.Transactions" />
     <Reference Include="System.Configuration" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
```

Comment:
It seems that building the core library for netcoreapp2.0 is unnecessary.

Is leaving off the `System.Reflection.Emit` package references worth it for the increased bulk in the NuGet package?